plugins/CoreHome/CoreHome.php

```diff
@@ -164,7 +165,7 @@ public function getJsFiles(&$jsFiles)
         $jsFiles[] = "node_modules/jquery.scrollto/jquery.scrollTo.min.js";
         $jsFiles[] = "node_modules/sprintf-js/dist/sprintf.min.js";
         $jsFiles[] = "node_modules/mousetrap/mousetrap.min.js";
-        $jsFiles[] = 'node_modules/angular/angular.min.js';
+        $jsFiles[] = Development::isEnabled() ? 'node_modules/angular/angular.js' : 'node_modules/angular/angular.min.js';
```

Comment:
Found myself having to debug angularjs quite often w/ the unminified code, so I thought I'd just add this.